internal/state/state.go

```diff
@@ -78,12 +76,8 @@ var OnNewMemberHook func(state *State) error
 var ReloadClusterCert func() error
 
 // Cluster returns a client for every member of a cluster, except
-// this one, with the UserAgentNotifier header set if a request is given.
-func (s *State) Cluster(r *http.Request) (client.Cluster, error) {
-	if r != nil {
-		r.Header.Set("User-Agent", request.UserAgentNotifier)
-	}
-
+// this one, with the UserAgentNotifier header set if isNotification is true.
```

Comment:
As the header isn't set anymore in this function should the second line of the comment be removed?

Reply:
It's not set in the function but it returns a client that applies it to all requests it makes. I'll clarify the comment.